### DISCUSSION TITLE
solaredge-modbus-multi work with the SE10K-RWB48BFN4 /BUI / 13.8 kWh battery ?

### DISCUSSION THREAD

**Discussion by hno2fe** [WillCodeForCats/solaredge-modbus-multi#494] [General]
### Describe the bug

Does the solaredge-modbus-multi work with the SE10K-RWB48BFN4 and the backup interface BUI and a 13.8 kWh battery, for me the values are only displayed correctly in the app in the portal & Modbus the values are also implausible.

### Expected behavior

![2024-01-06_12h03_39](https://github.com/WillCodeForCats/solaredge-modbus-multi/assets/145226308/a4424ef1-9f14-487d-95a4-17239acf7f72)
![2024-01-06_12h06_40](https://github.com/WillCodeForCats/solaredge-modbus-multi/assets/145226308/31211ae4-0972-4b90-9a4e-76f87d9cdb53)

real waren es aber   

![WhatsApp Image 2024-01-06 at 12 08 06](https://github.com/WillCodeForCats/solaredge-modbus-multi/assets/145226308/63a85a2f-e844-4e6a-8889-cc09d5473e13)

### Diagnostic File

Keine Ahnung wie das geht 

### Home Assistant Version

2024.1.1

### solaredge-modbus-multi Version

v2.4.8-pre.3  and earlier 

### Installation Type

HAOS

### Read the Instructions

- [X] I have read https://github.com/WillCodeForCats/solaredge-modbus-multi/wiki
- [X] My configuration follows https://github.com/WillCodeForCats/solaredge-modbus-multi/wiki/Configuration
- [X] This issue is not about a template error (template errors should not be reported as an issue, read https://github.com/WillCodeForCats/solaredge-modbus-multi/wiki/Template-Design-Notes)

### Additional Context

.

**Comment by WillCodeForCats** [comment]
All SolarEdge inverters have modbus option as far as I'm aware.
